### PR TITLE
docs: initial content for the Connectity class

### PR DESCRIPTION
**Pull request by Ombuweb**

### CODE REVIEW

**Comment by wSedlacek** [comment]
@Ombuweb I did some cleanup to align this with the current `Device` docs.
https://github.com/Ombuweb/docs/pull/1

If these changes are cool with you let's get them merged to your branch and get this added in.

**Comment by wSedlacek** [comment]
@rigor789 Can I get a once over to check nothing was missed?

File: content/guide/core/connectivity.md [hunk 1]
```diff
@@ -0,0 +1,138 @@
+---
+title: Connectivity
+description: Get the current device network connection type and monitor changes in the connection type.
+---
+
+The `Connectivity` class (singlton) abstracts the native API responsible for providing information about the network's connection type and availability.
```
Comment (SeanKelly369) [inline content/guide/core/connectivity.md]:
```suggestion
The `Connectivity` class, created as a singleton, abstracts the native API responsible for providing information about the network's connection type and availability.
```

File: content/guide/core/connectivity.md [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+if (connectionType) { // `Connectivity.connectionType.none`` is `0` so truthiness can be used to determine if the device is connected to any type of network
+  fetch('https://httpbin.org/get')
+    .then((response) => reponse.text())
+    .then((result) => console.log(`Fetched ${result} with ${connectionType}`));
+} else {
+  console.log("Not connected to a network.")
+}
+```
+
+Conditional behavior can also easily be added for certian types of connections using the `Connectivity.connectionType` enum.
```
Comment (SeanKelly369) [inline content/guide/core/connectivity.md]:
```suggestion
Conditional behavior can easily be added for certain types of connections using the `Connectivity.connectionType` enum.
```

File: content/guide/core/connectivity.md [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+            console.log("Connected to vpn network");
+            break;
+        default:
+            console.log("Not connected to any network");
+            break;
+    }
+});
+```
+
+If you would like to stop monitoring connecivity changes call [stopMonitoring()](#stopMonitoring)
```
Comment (SeanKelly369) [inline content/guide/core/connectivity.md]:
```suggestion
If you wish to halt the monitoring of connectivity changes, use the [stopMonitoring()](#stopMonitoring) function.
```

**Review by NathanWalker — APPROVED** [review]

**Pull request merged**